taqo_client/pubspec.yaml

```diff
@@ -89,6 +89,8 @@ flutter:
   #  - images/a_dot_ham.jpeg
   assets:
     - assets/
+    - ../LICENSE
```

Comment:
Nice, this way we don't need to copy the licenses into the assets folder during the build time.